### PR TITLE
Add chips logic and styles

### PR DESCRIPTION
**Pull request by NCMoseley**

### CODE REVIEW

File: js/components/Chips/Chips.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  ScrollView
+} from "react-native";
+
+import { styles } from "./styles";
+
+const Chips = ({
```
Comment (siddharthparmar7) [inline js/components/Chips/Chips.js:13]: Add propTypes for the props that you are receiving.

File: js/components/Chips/ChipsContainer.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import update from "immutability-helper";
+import Chips from "./Chips";
+
+import { styles } from "./styles";
+
+export default class ChipsContainer extends Component {
+  constructor() {
```
Comment (siddharthparmar7) [inline js/components/Chips/ChipsContainer.js]: if you are receiving props don't forget to pass "props" in your constructor and in your super.

File: js/components/Chips/ChipsContainer.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        max={max}
+        inputRef={input => {
+          this.TextInput = input;
+        }}
+      />
+    );
+  }
+}
+ChipsContainer.propTypes = {
+  chips: PropTypes.array,
```
Comment (siddharthparmar7) [inline js/components/Chips/ChipsContainer.js]: add .isRequired, if you can not do that then add default propType for all your props

File: js/components/Chips/styles.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { StyleSheet, Dimensions } from "react-native";
+import { colors, typography } from "../../config/styles";
```
Comment (siddharthparmar7) [inline js/components/Chips/styles.js]: I don't see you are using colors, so remove it please

File: js/components/Chips/styles.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+const { height, width } = Dimensions.get("window");
```
Comment (siddharthparmar7) [inline js/components/Chips/styles.js]: I don't see you are using height so remove it, please.

File: js/screens/Profile/Profile.js [hunk 6]
```diff
@@ -1,18 +1,50 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
```
Comment (siddharthparmar7) [inline js/screens/Profile/Profile.js]: remove importing propTypes if you are not using it.

File: js/screens/Profile/styles.js [hunk 7]
```diff
@@ -1,7 +1,84 @@
-import { StyleSheet } from "react-native";
+import { StyleSheet, Dimensions } from "react-native";
+import { colors, typography } from "../../config/styles";
+
+const { height, width } = Dimensions.get("window");
```
Comment (siddharthparmar7) [inline js/screens/Profile/styles.js]: Remove height if you are not using it.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Don't forget to assign yourself to this issue.
Also, please resolve your branch conflict. 
Thank you!

**Comment by NCMoseley** [comment]
Ok, looking good.

File: js/screens/Profile/ProfileContainer.js [hunk 8]
```diff
@@ -18,5 +18,7 @@ class ProfileContainer extends Component {
     return <Profile user={this.state.user} />;
   }
 }
-
+ProfileContainer.propTypes = {
```
Comment (siddharthparmar7) [inline js/screens/Profile/ProfileContainer.js]: You are not receiving any props for ProfileContainer so you don't need to check for propTypes. Also, don't import it.

File: js/screens/Profile/styles.js (hunk 7 above)
Comment (siddharthparmar7) [reply js/screens/Profile/styles.js]: remove height if you are not using it.

File: App.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Nodal - React Native App
+ * https://github.com/facebook/react-native
```
Comment (siddharthparmar7) [inline App.js]: while you are at it, remove line 3 and replace it with your project's GitHub link.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you! 

make these small changes soon and let me know so I can approve it.

**Comment by NCMoseley** [comment]
: |

**Review by siddharthparmar7 — APPROVED** [review]
Thank you! Nate!

**Pull request merged**